aspnetcore/migration/fx-to-core/examples/overview.md

```diff
@@ -2,11 +2,10 @@
 title: Learn to upgrade from ASP.NET MVC and Web API to ASP.NET Core MVC
 description: Learn how to upgrade an ASP.NET MVC Framework or Web API project to ASP.NET Core MVC
 author: rick-anderson
```

Comment:
```suggestion
author: wadepickett
```
I have to change the author meta data in order to receive github issues on the topic, so changing this too.